### ISSUE TITLE
Don't display an empty UV when attaching a resource with no FileSet

### ISSUE THREAD

**Issue by christinach** [pulibrary/figgy#2825]

**Comment by christinach** [comment]
This issue exists also with the scanned map. If a user attaches or creates a child that has a nil thumbnail_id then the parent scanned map will display an empty viewer as well.

**Comment by christinach** [comment]
In the case of the scanned map though if the users attaches/creates a 2nd, 3rd, .. child with a thumbnail_id then based on the thumbnail_builder#nearest_member, it will find the nearest child with a thumbnail_id <> nil and will display it in the viewer. 

In the case of the coin where we don't have nested coins, even if the members number is > 1 and one of them has a thumbnail_id <> nil,  the thumbnail_builder#nearest_member returns members = [].
https://github.com/pulibrary/figgy/blob/master/app/services/manifest_builder/thumbnail_builder.rb#L31

**Comment by christinach** [comment]
In my local if I run for numismatic_issue.id: 'dbe5e309-6ecb-41d9-a9ec-f37914cfd2bf' 
```
"SELECT member.* FROM orm_resources a, jsonb_array_elements(a.metadata->'member_ids') WITH ORDINALITY AS b(member, member_pos) JOIN orm_resources member ON (b.member->>'id')::UUID = member.id WHERE a.id = 'dbe5e309-6ecb-41d9-a9ec-f37914cfd2bf' AND member.internal_resource = 'Coin' ORDER BY b.member_pos"
``` then I take the same result (coin-15) which has no thumbnail_id so the nearest_member will return []

**Comment by christinach** [comment]
disregard the comment above. I was only querying for the first result the nearest_path works just fine with the issues. For some reason in my local environment I was able to reproduce the first comment I added consistently but I'm not able any more; which means that if the user attaches/creates a coin with an image and a coin with no image the issue will be able to find the thumbnail_id from the coin that has an image based on the nearest_path.
